src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,36 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.dao.BookDaoImpl;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        Book firstBook = new Book("Shantaram", BigDecimal.valueOf(548.60));
+        Book secondBook = new Book("The Godfather", BigDecimal.valueOf(350.50));
+        Book thirdBook = new Book("The Silmarillion", BigDecimal.valueOf(685.00));
+        BookDao bookDao = (BookDaoImpl) INJECTOR.getInstance(BookDao.class);
+        //inserting books to DB
+        bookDao.create(firstBook);
+        bookDao.create(secondBook);
+        bookDao.create(thirdBook);
+        //find by id
+        Long findId = 1L;
+        Book bookById = bookDao.findById(findId).orElseThrow(()
```

Comment:
id it’s autoincrement value and you can’t be sure that book with id 1 exists. So you should create a variable when you create book and use id of this object 
```
firstBook = bookDao.create(firstBook);
Book bookById = bookDao.findById(firstBook.getId())...

```